api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -419,7 +419,7 @@ private Map<Long, Double> getFreeTierWorkspaceCostsFromBQ(List<WorkspaceCostView
    *
    * <ol>
    *   <li>The workspace is active
-   *   <li>The workspace is deleted and either of the following is true
+   *   <li>The workspace is deleted within the past 6 months and either of the following is true
```

Comment:
wording: I'd prefer "any" here instead of "either", and we should move "either" to replace "1 of these cases" above